cmd/up.go

```diff
@@ -356,7 +356,30 @@ func (up *UpContext) devMode(isRetry bool, d *appsv1.Deployment, create bool) (b
 		}
 	}
 
-	p, err := pods.GetDevPod(up.Context, up.Dev, up.Client, create)
+	reporter := make(chan string)
+	go func() {
+		long := false
+		message := ""
+		ticker := time.NewTicker(1000 * time.Millisecond)
+		for {
+			select {
+			case <-ticker.C:
+				if long && message != "" {
+					spinner.update(fmt.Sprintf("%s...", message))
+				}
+				long = true
+			case message = <-reporter:
```

Comment:
Since we are waiting for a second, a lot of messages could come through, and we are not going to log them all.  Would it make sense to discard all the messages in the channel except for the last one when assigning to `message`?